modules/fundamental/src/advisory/endpoints/mod.rs

```diff
@@ -139,6 +140,7 @@ pub async fn delete(
         match rows_affected {
             0 => Ok(HttpResponse::NotFound().finish()),
             1 => {
+                fetched.delete_source_document(&tx).await?;
```

Comment:
Can't we do a "cascade delete"?

Reply:
No (or at least I didn't find a way to do it) as the relationship is reversed. We can add cascade to delete advisory when source document is deleted, but there's no reference from source document table to sbom/advisory tables.

Reply:
Ah, ok.